### ISSUE TITLE
[BUG] mct:IconTintColorBehavior doesn't work on windows<image> or <imagebutton> 

### ISSUE THREAD

**Issue by TheLankyScottishNerd** [CommunityToolkit/Maui#986]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Description

Using tint color to change the icon / image colours doesn't apply when deployed to windows.

an example project has been linked which shows two images, one of which is an imagebutton, with the behaviour attached and no visible changes.

<img width="712" alt="image" src="https://user-images.githubusercontent.com/8051530/218322617-fe052dc0-9015-4fee-b6c1-b30a1833ac59.png">

Works ok when deployed to iOS or android.
<img width="202" alt="image" src="https://user-images.githubusercontent.com/8051530/218322582-f2870af6-a1eb-4280-8fd9-6d022ba1a162.png">

### Expected Behavior

Using tint color should change the icon / image colours

### Steps To Reproduce

1. Create new MAUI App.
2. Add community toolkit and namespaces to project.
3. Add icon tint behaviour to the dotnet bot image.
4. Run on 3 platforms and observe bug.

### Link to public reproduction project repository

https://github.com/TheLankyScottishNerd/IconTintWinUI

### Environment

```markdown
- .NET MAUI CommunityToolkit: 4.0.0
- OS: Microsoft Windows 11 Pro - 10.0.22621 Build 22621
- .NET MAUI: 7.0.52
```

### Anything else?

Someone mentioned that this isn't supported on Windows... I don't seem to see that documented anywhere, though.

**Comment by pictos** [comment]
Hey @TheLankyScottishNerd, as I mentioned on the MAUI repo, this isn't an issue. We don't support TintColor on Windows, as you can see here.

https://github.com/CommunityToolkit/Maui/blob/0a6638b15cd3740b2021e6aef75b2eba2aecab20/src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.shared.cs#L8-L9

**Comment by TheLankyScottishNerd** [comment]
Hey @pictos, please accept my apologies. I did scratch my head for a while and did everything except jump into the actual code here in the repo or my project, so again I apologise.

Can we add an addition to the docs for this limitation? I am very new to MAUI and I didn't get any compile errors or any warnings in VS. 

I checked the docs here: https://github.com/MicrosoftDocs/CommunityToolkit/blob/main/docs/maui/behaviors/icon-tint-color-behavior.md and its not mentioned that this doesn't work and the link to the source goes to a 404.

Would a page in the docs which details the features working on which platforms for the toolkit be beneficial? Also, I wouldn't mind contributing to the docs, if at all possible that is.

Thanks again 👍🏻

**Comment by bijington** [comment]
> Hey @pictos, please accept my apologies. I did scratch my head for a while and did everything except jump into the actual code here in the repo or my project, so again I apologise.
> 
> Can we add an addition to the docs for this limitation? I am very new to MAUI and I didn't get any compile errors or any warnings in VS.
> 
> I checked the docs here: https://github.com/MicrosoftDocs/CommunityToolkit/blob/main/docs/maui/behaviors/icon-tint-color-behavior.md and its not mentioned that this doesn't work and the link to the source goes to a 404.
> 
> Would a page in the docs which details the features working on which platforms for the toolkit be beneficial? Also, I wouldn't mind contributing to the docs, if at all possible that is.
> 
> Thanks again 👍🏻

I have opened a PR to fix the path and add a note that Windows is not supported. https://github.com/MicrosoftDocs/CommunityToolkit/pull/227

Thanks for pointing this out.

**Comment by TheLankyScottishNerd** [comment]
Thanks @bijington,

Do you have any guidance/guidelines on how someone can contribute similarly if they find anything like this? 

Would it be a case of open a PR as you have done or would this need to start as a discussion/issue first?

Appreciate all the work the team and community do and would love to give back somehow where possible.

**Comment by TheLankyScottishNerd** [comment]
Thanks @bijington,

You know, I've seen those buttons so many times... just never even registered to use them!

Thanks again!

**Comment by lothrop** [comment]
This is now fixed with #1182.
